specification/templates/.github/CODEOWNERS

```
@@ -1,5 +1,6 @@
-# Learn about CODEOWNERS file format:
 #  https://help.github.com/en/articles/about-code-owners
 #
 # Global Owners
-* @signalfx/TEAM-maintainers
+* @signalfx/gdi-TEAM-maintainers @signalfx/gdi-TEAM-approvers
+
+CODEOWNERS @signalfx/gdi-TEAM-maintainers
```

Nit: shouldn't it be 
```suggestion
.github/CODEOWNERS @signalfx/gdi-TEAM-maintainers
```
since the template is in the `.github` dir?

---

I think it is better as it is.
It will be applied to any file named `CODEOWNERS`. More: https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners#codeowners-syntax
And the `CODEOWNERS` file can be in multiple locations: https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners#codeowners-file-location